src/containerapp/azext_containerapp/custom.py

```diff
@@ -2637,3 +2641,76 @@ def list_environment_telemetry_otlp(cmd,
+    return result
+
+
+def delete_dotnet_component(cmd, dotnet_component_name, environment_name, resource_group_name, no_wait):
+    exising_dotnet_component = _get_dotnet_component_if_exists(cmd, dotnet_component_name, environment_name, resource_group_name)
+    if not exising_dotnet_component:
+        raise ValidationError("DotNet Component '{}' does not exist in environment '{}' in resource group '{}'.".format(dotnet_component_name, environment_name, resource_group_name))
+    raw_parameters = locals()
+    dotnet_component_decorator = DotNetComponentDecorator(
+        cmd=cmd,
+        client=DotNetComponentPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    logger.warning("Deleting DotNet Component '%s' in environment '%s' in resource group '%s'.", dotnet_component_name, environment_name, resource_group_name)
+    result = dotnet_component_decorator.delete()
+    if result is None:
+        logger.warning("Successfully deleted DotNet Component '%s' in environment '%s' in resource group '%s'.", dotnet_component_name, environment_name, resource_group_name)
+    return result
+
+
+def create_dotnet_component(cmd, dotnet_component_name, environment_name, resource_group_name, no_wait, dotnet_component_type=DOTNET_COMPONENT_RESOURCE_TYPE):
+    # Check if DotNet component already exists in environment
+    existing_dotnet_component = _get_dotnet_component_if_exists(cmd, dotnet_component_name, environment_name, resource_group_name)
+    if existing_dotnet_component:
+        raise ValidationError("DotNet Component '{}' already exists in environment '{}' in resource group '{}'. Please disable Aspire Dashboard by deleting the DotNet Component before creating a new one.".format(dotnet_component_name, environment_name, resource_group_name))
+    raw_parameters = locals()
+    dotnet_component_decorator = DotNetComponentDecorator(
+        cmd=cmd,
+        client=DotNetComponentPreviewClient,
+        raw_parameters=raw_parameters,
+        models=CONTAINER_APPS_SDK_MODELS
+    )
+    dotnet_component_decorator.construct_payload()
+    managed_environment = show_managed_environment(cmd, environment_name, resource_group_name)
+    default_domain = safe_get(managed_environment, "properties", "defaultDomain")
+    if not default_domain:
+        raise ValidationError("The containerapp environment '{}' does not have a default domain.".format(environment_name))
+    aspire_dashboard_url = f"https://{dotnet_component_name}.ext.{default_domain}"
+    logger.warning("Creating DotNet Component '%s' in environment '%s' in resource group '%s.", dotnet_component_name, environment_name, resource_group_name)
+    r = dotnet_component_decorator.create()
+    if r is not None:
+        logger.warning("Successfully created DotNet Component '%s' in environment '%s' in resource group '%s'. Access your Aspire Dashboard at %s.", dotnet_component_name, environment_name, resource_group_name, aspire_dashboard_url)
```

Comment:
I did not see this message in your demo... I think it got hidden by the ARM payload that got returned at the end of the operation. Is there any way we could have this displayed after the payload is printed out?

Reply:
This message is getting printed out before returning the payload. Don't think there's a good way to print this message after returning the payload to customer. 

As an alternative, we could just print this message and not return the payload at all.